source/Makefile.Objects

```diff
@@ -222,6 +222,7 @@ OBJS_ESOLVER_LCAO=esolver_ks_lcao.o\
       esolver_ks_lcao_tddft.o\
       esolver_ks_lcao_tmpfunc.o\
       io_npz.o\
+      orb_paramter_init.o\
```

Comment:
put this file into gint module.

Reply:
This function must use the GloablC parameter, so it is more appropriate to place it in the ORB folder.